pkg/serverless/metrics/enhanced_metrics.go

```diff
@@ -526,6 +527,14 @@ func generateTmpEnhancedMetrics(args generateTmpEnhancedMetricsArgs) {
 		SampleRate: 1,
 		Timestamp:  args.Time,
 	})
+	args.Demux.AggregateSample(metrics.MetricSample{
+		Name:       tmpFreeMetric,
+		Value:      args.TmpMax - args.TmpUsed,
+		Mtype:      metrics.DistributionType,
+		Tags:       args.Tags,
+		SampleRate: 1,
+		Timestamp:  args.Time,
+	})
```

Comment:
Apologies for not understanding this better, but what's the reason behind adding this metric? Customers can already technically produce this data on their own using a bit of metric math.